services/reference/linea/quickstart.md

```diff
@@ -15,7 +15,23 @@ Don't have an Infura account? Sign up for our free plan and start using the Line
 
 ## Prerequisites
 
-Ensure you have an [API key](../../../../developer-tools/dashboard/get-started/create-api/) with the Linea network enabled.
+- Ensure you have an [API key](../../../../developer-tools/dashboard/get-started/create-api/) with the Linea network enabled.
+- For calls that require gas, topup your funds.
```

Comment:
top up  is the verb I think

Reply:
I suggested to remove this completely in my comment. But if we were to keep this, even "top up" is a bit unclear – it should be something like "For calls that require gas, make sure you have enough funds."

Reply:
Yeah there was no clear mapping Infura:MM for this one. IMO this is not the place, rather -- I think there should be a link to the "Get testnet ETH and tokens"  page from the index page's "See also" forEach service --> let me know if I should raise issue for that.

Reply:
Not high priority, but that's fine ^